shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -360,10 +360,6 @@ public void setInitialRoute(String route) {
     navigationChannel.setInitialRoute(route);
   }
 
-  public void pushRoute(String route) {
```

Comment:
Is this a breaking change?

Reply:
I am not sure who is using this API, and if there is a way to access this API at all. I can restore it if this is needed though

Reply:
This is the old embedder v1 API. I think there were ongoing efforts to remove it but the person(s) working on that are no longer on the team.

Reply:
If it's possible, you should put this back, Otherwise LGTM